esx_service/vmodl/vmodl_test.py

```diff
@@ -573,12 +588,31 @@ def test_add_privilege_already_exists(self):
         with self.assertRaises(vim.fault.AlreadyExists):
             self.tenantMgr.AddPrivilege(tenant, privilege)
 
-    def test_add_privilege_invalid_arg(self):
+    def test_add_privilege_invalid_datastore(self):
+        # Create a tenant
+        tenant = self.tenantMgr.CreateTenant(name=TENANT_NAME, description=TENANT_DESC)
+
+        # Create a privilege with invalid datastore
+        privilege = vim.vcs.storage.DatastoreAccessPrivilege()
+        privilege.datastore = DS_NOT_EXIST
+        privilege.allow_create = False
+        privilege.volume_max_size = 1024
+        privilege.volume_total_size = 2048
+
+        # Add the privilege to the tenant
+        with self.assertRaises(vmodl.fault.InvalidArgument):
+            self.tenantMgr.AddPrivilege(tenant, privilege)
+
+    def test_add_privilege_invalid_volume_size(self):
```

Comment:
Please don't :-) , it will be totally unreadable. 
The current one is good enough IMO, especially if you add a good docstring. 

@lipingxue - feel free to offer a better name though, with a reasonable length()